src/test/java/com/google/devtools/build/lib/skyframe/BUILD

```diff
@@ -60,7 +60,7 @@ java_test(
             exclude = ["MacOSXFsEventsDiffAwarenessTest.java"],
         ),
     }),
-    flaky = 1,
+    shard_count = 20,
```

Comment:
Is it no longer flaky?

Reply:
```
$ bazel test --runs_per_test=1000 //src/test/java/com/google/devtools/build/lib/skyframe:SkyframeTests
...
Target //src/test/java/com/google/devtools/build/lib/skyframe:SkyframeTests up-to-date:
  bazel-bin/src/test/java/com/google/devtools/build/lib/skyframe/SkyframeTests.jar
  bazel-bin/src/test/java/com/google/devtools/build/lib/skyframe/SkyframeTests
INFO: Elapsed time: 11574.767s, Critical Path: 463.55s
INFO: 21121 processes: 20844 linux-sandbox, 277 worker.
INFO: Build completed successfully, 21125 total actions
Test cases: finished with 42 passing and 0 failing out of 42 test cases

Executed 1 out of 1 test: 1 test passes.
INFO: Build completed successfully, 21125 total actions
```

It's not flaky on Linux. Is there a way to use the CI system to run the same check for Mac and Windows?

Reply:
Not that I know of. Thanks for checking its flakiness!

Let's remove the attribute and restore if the test is indeed flaky.